tasks/main.yml

```diff
@@ -40,9 +40,10 @@
   tags:
     - nrpe
     - install
-  package:
+  yum:
     name: "{{ item }}"
     state: present
+    enablerepo: las_EPEL7_epel7
```

Comment:
I don't think we want this in the role. This should be in a group var instead (too specific to be in the role)